README.md

```diff
@@ -1,34 +1,67 @@
-# Chime SDK UI Component Library
+# Chime SDK React Components Library
+<a href="https://www.npmjs.com/package/amazon-chime-sdk-component-library-react"><img src="https://img.shields.io/npm/v/amazon-chime-sdk-component-library-react?style=flat-square"></a>
+<a href="https://github.com/aws/amazon-chime-sdk-component-library-react"><img src="https://github.com/aws/amazon-chime-sdk-component-library-react/workflows/CI%20Workflow/badge.svg"></a>
+
+[React Library Documentation](https://aws.github.io/amazon-chime-sdk-component-library-react/)
+[Chime SDK JavaScript Library](https://github.com/aws/amazon-chime-sdk-js/)
+
+The Amazon Chime SDK makes it easy to add collaborative audio calling, video calling, and screen share features to web applications by using the same infrastructure services that power millions of Amazon Chime online meetings.
+
+The Amazon Chime SDK React Component Library supplies client-side state management and reusable UI components for common web interfaces used in audio and video conferencing applications, including: video tile grids, microphone activity indicators, and call controls. All components come with a simple, modern design, and can be used as-is or restyled with a custom theme. In addition to UI components, the library leverages Reacts' state management tools such as Providers and Hooks to connect to the Amazon Chime SDK and pass data to the UI layer, simplifying state synchronization so that developers can concentrate on building engaging experiences.
```

Comment:
Should we mention specifically "Amazon Chime SDK for Javascript" in the last line?

For e.g: .... Amazon Chime SDK for Javascript and pass data to the UI layer, ..